Woof/Woof/Helper/AppStyle/FontSize.swift

```diff
@@ -0,0 +1,13 @@
+import SwiftUI
+
+extension AppStyle {
+    /// Provides the font sizes used within the application.
+    enum FontSize {
+        /// Size of font uses for titles and headings.
+        static let heading = 22.0
+        /// Size of font uses as main font size.
+        static let body = 18.0
+        /// Size of font uses for additional elements.
```

Comment:
nit: I see no info about this size at [the link](https://github.com/ios-course/ironfoudation-team-project/wiki/App-style-guidelines#fonts-and-sizes) provided above.

Reply:
Added to App guideline  missing values.